api_tests/src/benchmark_baseline.spec.ts

```diff
@@ -0,0 +1,126 @@
+/*
+With Lemmy 0.18.3 and earlier, performance has been a big concern.
+Logging basic expectations of response times is the purpose of this module.
+*/
+jest.setTimeout(120 * 1000);
```

Comment:
Running benchmarks in this way is not going to work well because performance of dev/ci machines is unpredictable. Besides the workload and database content is too different from actual production instances.